app/src/main/java/tech/ula/utils/Logger.kt

```diff
@@ -85,12 +92,14 @@ class SentryLogger : Logger {
                 .withMessage(message)
                 .withLevel(Event.Level.ERROR)
         Sentry.capture(event)
+        Log.e("ILLEGAL_STATE", message)
     }
 
     override fun sendEvent(message: String) {
         val event = EventBuilder()
                 .withMessage(message)
                 .withLevel(Event.Level.ERROR)
         Sentry.capture(event)
+        Log.e("EVENT", message)
```

Comment:
Looks like this is causing a failing test, complaining that `Log.e` hasn't been mocked in `FilesystemListViewModelTest`